Source/OIDError.h

```diff
@@ -118,6 +118,12 @@ typedef NS_ENUM(NSInteger, OIDErrorCode) {
           request in the default browser.
    */
   OIDErrorCodeBrowserOpenError = -10,
+
+  /*! @brief Indicates a problem when trying to refresh the tokens.
+   */
+  OIDErrorCodeTokenRefreshError = -11,
```

Comment:
Conflicts with [change](https://github.com/openid/AppAuth-iOS/pull/22/files#diff-6d5d553d8f414d088c8019134942c79bR137) introduced by #22, which should take precedence?